sentences/ro/_common.yaml

```diff
@@ -54,14 +54,15 @@ expansion_rules:
   este: "(e | este)"
   modifica: "(schimb(ă|a) | modific(ă|a) | seteaz(ă|a))"
   in: "((î|i)n)"
-  din: "(din | (î|i)n | pentru | de la)"
+  din: "(din | (î|i)n | pentru | de la | de pe)"
   porneste: "(start | porne(ș|s)te | deschide | aprinde | activeaz(ă|a))"
   opreste: "(stop | opre(ș|s)te | (î|i)nchide | stinge | dezactiveaz(ă|a))"
   seteaza: "(seteaz(ă|a) | pune | adjusteaz(ă|a) | schimb(ă|a) | modific(ă|a))"
-  lumina: "(luminile | lumina | becul | becurile)"
+  lumina: "(lumin(a|ile) | becu(l|rile) | aplic(a|ele) | lampa | lămpile | veioz(a|ele) | ledu(l|rile))"
```

Comment:
Sintagma `<lumina>` e folosita pentru a defini domeniul `light`, nu neaprat toate tipurile de corpuri de iluminat.

Problema e ca daca folosim toate aceste cuvinte, HA ar stinge toate luminile in momentul in care primeste comanda `stinge veioza`, ceea ce mi se pare contraintuitiv.

"Aprinde lumina" este in mod clar universal valabil pentru orice tip de corp de iluminat. Nu stiu exact daca "becul/becurile" se incadreaza in aceeasi categorie.